src/pages/Discover/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react';
+import Card from '../../components/Card';
+import './styles.css';
+
+export default function DiscoverPage() {
+  const [places, setPlaces] = useState([]);
+
+  useEffect(() => {
+    async function getPlaces() {
+      const url =
+        'https://raw.githubusercontent.com/manypossibles/designops/master/assets/data/en/data.json';
```

Comment:
Put the url in a constant. Make a folder called `utils` and file called `config.js` where you can do `export const VISIT_AARHUS_API_URL='blalba'`